testutil/race_off.go

```diff
@@ -0,0 +1,7 @@
+//go:build !race
+
+package testutil
+
+const (
+	RaceEnabled = false
```

Comment:
Suggestion: We could unexport this const and add a function that returns it, this will allow us to document it in one place.